x-pack/plugins/enterprise_search/public/applications/workplace_search/components/overview/statistic_card.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { EuiCard, EuiFlexItem, EuiLink, EuiTitle } from '@elastic/eui';
+
+import { useRoutes } from '../shared/use_routes';
+
+interface IStatisticCardProps {
+  title: string;
+  count?: number;
+  actionPath?: string;
+}
+
+export const StatisticCard: React.FC<IStatisticCardProps> = ({ title, count = 0, actionPath }) => {
+  const { getWSRoute } = useRoutes();
+
+  const card = (
+    <EuiFlexItem>
+      <EuiCard
+        className={actionPath ? 'euiCard--isClickable' : ''}
```

Comment:
If possible we should prefer using EuiCard's default `href`/`onClick` behavior over mimicking it by calling their classes

Reply:
7ba1184